### PR TITLE
Fixed failing build on xcode 10.1 

### PR DESCRIPTION
**Pull request by dtrabo**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Fixed failing build on xcode 10.1+ with "Failed to verify bitcode while exporting archive..."

## Related PRs or issues

[AB#58471](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/58471)

### CODE REVIEW

File: Sasquatch/Config/Sasquatch.xcconfig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include "../../Config/App.xcconfig"
 
 PRE_XCODE_10_1_ARCHS = armv7 armv7s arm64
-ARCHS[sdk=iphoneos*] = $(PRE_XCODE_10_1_ARCHS) arm64e
+ARCHS[sdk=iphoneos*] = $(PRE_XCODE_10_1_ARCHS)
```
Comment (clpolet) [inline Sasquatch/Config/Sasquatch.xcconfig]: I would set arm64e on Debug builds so that we can still test it.

Comment (jaelim-ms) [reply Sasquatch/Config/Sasquatch.xcconfig]: Can we clean up PRE_XCODE_10_1_ARCHS? We dropped Xcode 8/9 and officially use 10.1 for build. We should be able to remove this. @clpolet any concerns?

Comment (clpolet) [reply Sasquatch/Config/Sasquatch.xcconfig]: We need to keep it for Xcode 10.0. But you can remove all the lines referencing older Xcode versions so lines containing:

```
ARCHS[sdk=iphoneos9.*] = $(PRE_XCODE_10_1_ARCHS)
ARCHS[sdk=iphoneos10.*] = $(PRE_XCODE_10_1_ARCHS)
ARCHS[sdk=iphoneos11.*] = $(PRE_XCODE_10_1_ARCHS)
```

Comment (clpolet) [reply Sasquatch/Config/Sasquatch.xcconfig]: But can be done in a separate PR. We will also have to get rid of the `PRE_XCODE_10_1_WATCH_ARCHS`, arm64_32 came with Xcode 10.

**Pull request merged**